userportaldatamodel/notification_log.py

```diff
@@ -27,17 +27,15 @@
 class NotificationLog(Base):
     __tablename__ = "notification_log"
 
-    messageid = Column(Integer, primary_key=True, nullable=False, autoincrement=True) 
-    message = Column(String)
+    id = Column(Integer, primary_key=True, nullable=False, autoincrement=True) 
+    message = Column(String, nullable=False)
```

Comment:
I think this table could just be merged into the other table since every row here will be a row in the other table. Also there is a table called message in this repo it is probably better to reference that table as message rather than have this column

Reply:
@paulmurdoch19 One table is supposed to be just the messages, while the the second is a many to one with the users. To keep track of which user saw which message. So the same message will be seen by more users.